### ISSUE TITLE
entrypoints vs entryPoints mismatch

### ISSUE THREAD

**Issue by Rikarin** [containous/traefik#4768]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [ ] No

### What did you expect to see?
I'm using IngressRoute in kubernetes.

When I use "entrypoints"
```
spec:
  entrypoints:
    - websecure
```
I cannot see entrypoints in /api/rawdata but when I use "entryPoints" instead, I can see it in /api/rawdata but I get 404 not found.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
(paste your output here)
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
# (paste your configuration here)
```

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

```
(paste your output here)
```

**Comment by ldez** [comment]
Hello, it's already fixed in #4764

**Comment by Rikarin** [comment]
OK, thanks.
